shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/domain/platform/VersionCode.kt

```diff
@@ -0,0 +1,11 @@
+package co.nimblehq.blisskmmic.domain.platform
+
+interface VersionCode {
+    val versionCode: String
+    val buildNumber: String
+}
+
+expect class VersionCodeImpl(): VersionCode {
+    override val versionCode: String
+    override val buildNumber: String
+}
```

Comment:
we should create separated files for the `interface` and the `implementation` class 🙏

Reply:
They both belong in the same `domain` layer. This is similar to how we declare UseCase.